src/app/views/route-search-results/route-search-results.component.html

```diff
@@ -1,41 +1,126 @@
 <app-nav-bar title="Results"></app-nav-bar>
 
 <div class="big-navigation-gap">
-  <div class="route-search-results__route-information">
-    <div>
+  <div class="route-search-results__route-information center-center">
+    <div class="center-center">
       <div>A</div>
       <div>B</div>
     </div>
-    <div>
-      <div>{{ routeS.getMainFrom(route) }}</div>
-      <div>{{ routeS.getMainTo(route) }}</div>
+    <div class="route-search-results__nowrap-hidden center-center">
+      <div>
+        <span>{{ routeS.getMainFrom(route) }}</span>
+        <span class="route-search-results__location-secondary">, {{ routeS.getSecondaryFrom(route) }}</span>
+      </div>
+      <div>
+        <span>{{ routeS.getMainTo(route) }}</span>
+        <span class="route-search-results__location-secondary">, {{ routeS.getSecondaryTo(route) }}</span>
+      </div>
```

Comment:
i generally really like how the design turned out :)

- [ ] the location names dont align correctly yet

<img width="118" alt="Screen Shot 2020-10-29 at 18 05 35" src="https://user-images.githubusercontent.com/16033930/97607735-a8947580-1a11-11eb-8da1-89e0ced327c8.png">

- [ ] could we replace the A and B with maybe the same circle as in the previous routes view?

- [ ] there exists no fade or dots at the end of the line to indicate a long name